daisy_workflows/image_build/debian/debian.wf.json

```diff
@@ -52,17 +47,10 @@
     "wait": {
```

Comment:
Our other build workflows create the image in GCE and don't export the image to GCS